moderation/__init__.py

```diff
@@ -1,3 +1,25 @@
-from moderation.register import ModerationManager
+default_app_config = "moderation.apps.ModerationConfig"
 
-moderation = ModerationManager()
+
+class _ModerationProxy(object):
+    """Proxy the ModerationManager()
+
+    It must not be created here for Django >= 1.8, because __init__ is invoked
+    too early, but must be present once a module registers its models.
+    """
+    _moderation = None
+
+    def _ensure_obj(self):
+        if _ModerationProxy._moderation is None:
+            from moderation.register import ModerationManager
+            _ModerationProxy._moderation = ModerationManager()
+
+    def __getattr__(self, attribute):
+        self._ensure_obj()
+        return getattr(_ModerationProxy._moderation, attribute)
+
+    def __setattr__(self, attribute, value):
+        self._ensure_obj()
+        return setattr(_ModerationProxy._moderation, attribute, value)
+
+moderation = _ModerationProxy()
```

Comment:
This is required because Django >= 1.7 imports the module at a stage where the app registry isn't complete yet. Importing `moderation.register` at that stage fails, so it must be postponed. Still, `moderation.moderation` cannot be declared in the `apps.py` file (where it belongs), because it is accessed in the tests before the app is ready.